### PR TITLE
Front-end UI

### LINKED ISSUE #8: Interface for checking a single person with a passport number
As the contact centre staff
I should be able to look up the information of a person by ID
And it returns their information 

 A / C
 - [x] Should be able to enter a passport number
 - [x] That person's information presented on the screen
 - [x] Should bring represent the design

https://docs.google.com/presentation/d/17wZvfOO0UTRuAE_XGEm_KBuY1XBdIAq0QgaZtVb_gpM/edit#slide=id.g47c1588cff_0_136

Dev tasks:
 - [ ] Data model for applicant
 - [ ] Model tests (rspec)
 - [ ]  Feature spec (rspec)
 - [ ] Controller tests, that you must be logged in

### PR DESCRIPTION
**Pull request by dlouise64**
closes #8

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,11 +10,27 @@
...
 
+  create_table "applicants", force: :cascade do |t|
+    t.string "first_name"
+    t.string "last_name"
+    t.string "passport_number"
+    t.string "email"
+  end
+
+  create_table "clients", force: :cascade do |t|
+    t.string "identity"
```
Comment (Br3nda) [inline db/schema.rb]: i think it should be the other way around. An identity belongs to a client, so this coloun shouldn't be there. Instead there's a `t.references :clients` on the identities table, and identities.rb has `belong_to :client`

**Comment by dlouise64** [comment]
@Br3nda @joris-rotteveel I am going to go through and clean up code climate

File: app/controllers/clients_controller.rb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        format.json { render json: @client.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /clients/1
+  # PATCH/PUT /clients/1.json
+  def update
+    respond_to do |format|
+      if @client.update(client_params)
```
Comment (Br3nda) [inline app/controllers/clients_controller.rb]: i don't think we need any of these -- we're only viewing data - not modifying it

File: app/javascript/bundles/HelloWorld/components/HelloWorld.jsx [hunk 3]
```diff
@@ -24,11 +24,11 @@ export default class HelloWorld extends React.Component {
   render() {
     return (
       <div>
-        <h3>
+        {/* <h3>
           Hello, {this.state.name}!
-        </h3>
+        </h3> */}
```
Comment (Br3nda) [inline app/javascript/bundles/HelloWorld/components/HelloWorld.jsx]: how about deleting these instead of comments

File: app/javascript/bundles/Identity/components/KnownIdentity.jsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+export default class KnownIdentity extends React.Component {
+  static propTypes = {
+    identities: PropTypes.array
+  };
+
+  render() {
+    const {
+      id,
+      last_name,
```
Comment (Br3nda) [inline app/javascript/bundles/Identity/components/KnownIdentity.jsx]: should be 
surname
first_name
second_name
third_name

File: app/views/movements/show.json.jbuilder [hunk 5]
```diff
@@ -0,0 +1 @@
+json.partial! "movements/movement", movement: @movement
```
[the remark this answers is not in the document]
Comment (Br3nda) [reply app/views/movements/show.json.jbuilder]: i don't think we need a jbuilder here

**Comment by Br3nda** [comment]
I'm adding te models here

File: app/controllers/clients_controller.rb [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  def index
+    @clients = Client.all
+  end
+
+  # GET /clients/1
+  # GET /clients/1.json
+  def show
+    @identities = Identity.where(client_id: params[:id])
+
+    @movements = Movement.where(client_id: params[:id]) if Movement.count > 0
```
[the remark this answers is not in the document]
Comment (Br3nda) [reply app/controllers/clients_controller.rb]: I think you want to look up a client instead
@client = Client.find(params[:id])

and then you'll have easy access like this:
@client.identities
@client.movements

File: app/controllers/application_controller.rb [hunk 7]
```diff
@@ -1,5 +1,4 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
-  before_action :authenticate_user!
```
Comment (Br3nda) [inline app/controllers/application_controller.rb]: Here you're turning off the requirement to be authenticated before you use this app. I think this is a mistake? !?

File: app/javascript/bundles/HelloWorld/components/HelloWorld.jsx [hunk 8]
```diff
@@ -24,11 +24,11 @@ export default class HelloWorld extends React.Component {
   render() {
```
Comment (Br3nda) [inline app/javascript/bundles/HelloWorld/components/HelloWorld.jsx]: is this file still needed?

File: app/javascript/packs/hello-world-bundle.js [hunk 9]
```diff
@@ -1,8 +1,10 @@
-import ReactOnRails from 'react-on-rails';
+import ReactOnRails from "react-on-rails";
```
Comment (Br3nda) [inline app/javascript/packs/hello-world-bundle.js]: do we still need this file?

File: app/views/clients/_form.html.erb [hunk 10]
```diff
@@ -0,0 +1,22 @@
+<%= form_with(model: client, local: true) do |form| %>
```
Comment (Br3nda) [inline app/views/clients/_form.html.erb]: i think this file isn't used

File: app/controllers/application_controller.rb (hunk 7 above)
Comment (Br3nda) [reply app/controllers/application_controller.rb]: i turned it back on

**Comment by Br3nda** [comment]
It's failing to deploy with this error:
```
       rake aborted!

       Sass::SyntaxError: File to import not found or unreadable: ./lists.scss.

       (sass):260
```

**Comment by Br3nda** [comment]
I got deployment working - i had to comment out the SCSS includes because they were making webpacker fail. I've made a issue to track that #99
